### DISCUSSION TITLE
Sponsored Products reports

### DISCUSSION THREAD

**Discussion by shajia313** [amzn/ads-advanced-tools-docs#133] [Q&A]
Regarding the form of obtaining various data from the advertising API, at present, it mainly summarizes all the data of the site through reporting or displays it daily. Whether it exists: the data of a certain asin/keyword/targeting in Sponsored Products can be directly obtained through the API

**Comment by elizafarley** [comment]
HI @shajia313, thanks for your question. If you want performance metrics for advertised ASINs, you would need to use the [advertised product report](https://advertising.amazon.com/API/docs/en-us/guides/reporting/v3/report-types#advertised-product-reports), which can be requested at either the daily or summary aggregation level, up to 65 days back from today. For keyword and product targeting, you can use the [targeting report](https://advertising.amazon.com/API/docs/en-us/guides/reporting/v3/report-types#targeting-reports).

There is currently no way to request metrics for a single ASIN/keyword/target, but you can filter by targetId/keywordId/asin in the relevant report. 

Let me know if that helps!

**Reply by shajia313** [reply]
Thank you, very helpful
